src/system/physical/button.cpp

```diff
@@ -40,17 +40,16 @@ void treat_button_pressed(const bool isButtonPressDetected,
                           const std::function<void(uint8_t)>& clickSerieCallback,
                           const std::function<void(uint8_t, uint32_t)>& clickHoldSerieCallback)
 {
-  buttonState.wasTriggered = false;
   buttonState.lastEventTime = millis();
   buttonState.sinceLastCall = buttonState.lastEventTime - buttonState.lastPressTime;
   buttonState.pressDuration = buttonState.lastEventTime - buttonState.firstHoldTime;
 
   // currently in long press status
   buttonState.isLongPressed = (buttonState.isPressed && buttonState.pressDuration > HOLD_BUTTON_MIN_MS);
 
-  // remove button clicked if last call was too long ago
-  if ((buttonState.sinceLastCall > RELEASE_TIMING_MS) ||
-      (buttonState.isLongPressed && buttonState.sinceLastCall > RELEASE_TIMING_MS / 2))
+  // remove button clicked if last call was too long ago (and an action is currently handled)
+  if (buttonState.wasTriggered and ((buttonState.sinceLastCall > RELEASE_TIMING_MS) or
```

Comment:
```suggestion
  if (buttonState.wasTriggered && ((buttonState.sinceLastCall > RELEASE_TIMING_MS) ||
```
let's be consistent on the use of `&&` and `||`

Reply:
By default in the code I try to use the AND, OR and NOT instead of && || and !, because it's really easy to misread/write and confuse everything with the binary equivalents (& | !)